inlong-manager/manager-dao/src/main/resources/mappers/SortClusterMapper.xml

```diff
@@ -0,0 +1,80 @@
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.apache.inlong.manager.dao.mapper.SortClusterMapper">
+    <resultMap id="SortTaskDTO" type="org.apache.inlong.common.pojo.sortstandalone.SortTaskDTO">
+        <result column="sort_task_name" jdbcType="VARCHAR" property="sortTaskName"/>
+        <result column="sort_consumer_group" jdbcType="VARCHAR" property="sortConsumerGroup"/>
+        <result column="sink_type" jdbcType="VARCHAR" property="sinkType"/>
+        <result column="data_node_name" jdbcType="VARCHAR" property="dataNodeName"/>
+    </resultMap>
+    <resultMap id="SortSinkParamsDTO" type="org.apache.inlong.common.pojo.sortstandalone.SortSinkParamsDTO">
+        <result column="name" jdbcType="VARCHAR" property="name"/>
+        <result column="type" jdbcType="VARCHAR" property="type"/>
+        <result column="ext_params" jdbcType="VARCHAR" property="extParams"/>
+    </resultMap>
+    <resultMap id="SortIdParamsDTO" type="org.apache.inlong.common.pojo.sortstandalone.SortIdParamsDTO">
+        <result column="inlong_group_id" jdbcType="VARCHAR" property="inlongGroupId"/>
+        <result column="inlong_stream_id" jdbcType="VARCHAR" property="inlongStreamId"/>
+        <result column="ext_params" jdbcType="VARCHAR" property="extParams"/>
+    </resultMap>
+
+    <select id="selectTasksByClusterName" resultMap="SortTaskDTO">
+        select
+            sort_task_name, sort_consumer_group, sink_type, data_node_name
+        from stream_sink
+        <where>
+            is_deleted = 0
+            <if test="clusterName != null and clusterName != ''">
+                and inlong_cluster_name = #{clusterName, jdbcType=VARCHAR}
+            </if>
+        </where>
+        group by
+            sort_task_name, sink_type, sort_consumer_group, data_node_name
+    </select>
+
+    <select id="selectSinkParamsByNameAndType" resultMap="SortSinkParamsDTO">
+        select
+            name, type, ext_params
+        from data_node
+        <where>
+            is_deleted = 0
+            <if test="dataNodeName != null and dataNodeName != ''">
+                and name = #{dataNodeName, jdbcType=VARCHAR}
+            </if>
+            <if test="type != null and type != ''">
+                and type = #{type, jdbcType=VARCHAR}
+            </if>
+        </where>
+    </select>
+    
+    <select id="selectIdParamsByTaskName" resultMap="SortIdParamsDTO">
+        select
+            inlong_group_id, inlong_stream_id, ext_params
+        from stream_sink
+        <where>
+            is_deleted = 0
+            <if test="taskName != null and taskName != ''">
+                and sort_task_name = #{taskName, jdbcType=VARCHAR}
+            </if>
```

Comment:
Why not select all stream_sink data?
It is too many times to execute SQL with sort_task_name.